packages/jupyter-chat/src/components/input/chat-input.tsx

```diff
@@ -167,92 +203,139 @@ export function ChatInput(props: ChatInput.IProps): JSX.Element {
     }
   }
 
-  // Set the helper text based on whether Shift+Enter is used for sending.
-  const helperText = sendWithShiftEnter ? (
-    <span>
-      Press <b>Shift</b>+<b>Enter</b> to send message
-    </span>
-  ) : (
-    <span>
-      Press <b>Shift</b>+<b>Enter</b> to add a new line
-    </span>
-  );
+  const horizontalPadding = props.area === 'sidebar' ? 1.5 : 2;
 
   return (
     <Box
       sx={props.sx}
       className={clsx(INPUT_BOX_CLASS)}
       data-input-id={model.id}
     >
-      <AttachmentPreviewList
-        attachments={attachments}
-        onRemove={model.removeAttachment}
-      />
-      <Autocomplete
-        {...chatCommands.autocompleteProps}
-        // ensure the autocomplete popup always renders on top
-        slotProps={{
-          popper: {
-            placement: 'top'
-          },
-          paper: {
-            sx: {
-              border: '1px solid lightgray'
-            }
-          },
-          listbox: {
-            sx: {
-              '& .MuiAutocomplete-option': {
-                padding: 2
-              }
-            }
-          }
+      <Box
+        sx={{
+          border: '1px solid',
+          borderColor: isFocused
+            ? 'var(--jp-brand-color1)'
+            : 'var(--jp-border-color1)',
+          borderRadius: 2,
+          transition: 'border-color 0.2s ease',
+          display: 'flex',
+          flexDirection: 'column',
+          overflow: 'hidden'
         }}
-        renderInput={params => (
-          <TextField
-            {...params}
-            fullWidth
-            variant="filled"
-            className={INPUT_TEXTFIELD_CLASS}
-            multiline
-            onKeyDown={handleKeyDown}
-            placeholder="Start chatting"
-            inputRef={inputRef}
-            sx={{ marginTop: '1px' }}
-            onSelect={() =>
-              (model.cursorIndex = inputRef.current?.selectionStart ?? null)
-            }
-            slotProps={{
-              input: {
-                ...params.InputProps,
-                className: INPUT_COMPONENT_CLASS
-              }
+      >
+        {attachments.length > 0 && (
+          <Box
+            sx={{
+              px: horizontalPadding,
+              pt: 1,
+              pb: 1
             }}
-            label={input.length > 2 ? helperText : ' '}
-          />
+          >
+            <AttachmentPreviewList
```

Comment:
Don't know why but I'm not able to open an attachment from the input.
It used to be possible AFAIK, and it is still possible for an attachment in the message (after sending it).

Reply:
Hmmm, that is working for me.